README.md

```diff
@@ -21,9 +21,22 @@ which supports:
 It includes micro-benchmark for primitive computation and communication benchmarking,
 and model-benchmark to measure domain-aware end-to-end deep learning workloads.
 
+> 🔴 __Note__:
+SuperBench is in the early pre-alpha stage for open source, and not ready for general public yet.
+If you want to jump in early, you can try building [`dev` branch](https://github.com/microsoft/superbenchmark/tree/dev) yourself.
+
 
 ## Installation
 
+### Using Docker (_Preferred_)
+
+```sh
+docker pull superbench/superbench:dev-cuda11.1.1
+docker run -it --rm \
+    --privileged --net=host --ipc=host --gpus=all \
+    superbench/superbench:dev-cuda11.1.1 bash
+```
+
 ### Using Python
 
 System requirements:
```

Comment:
Could we use headings format or bold?

#### System Requirements

#### Install SuperBench